--- readline-6.3/signals.c	2014-01-10 15:06:48.000000000 -0500
+++ readline-6.3.new/signals.c	2015-01-01 13:37:22.096225788 -0500
@@ -324,11 +326,7 @@
   struct sigaction act;
 
   act.sa_handler = handler;
-#  if defined (SIGWINCH)
-  act.sa_flags = (sig == SIGWINCH) ? SA_RESTART : 0;
-#  else
   act.sa_flags = 0;
-#  endif /* SIGWINCH */
   sigemptyset (&act.sa_mask);
   sigemptyset (&ohandler->sa_mask);
   sigaction (sig, &act, &old_handler);
